x-pack/plugin/sql/qa/src/main/resources/null.csv-spec

```diff
@@ -0,0 +1,60 @@
+//
+// Null expressions
+//
+
+dateTimeOverNull
+SELECT YEAR(CAST(NULL AS DATE)) d;
+
+d:i
+null
+;
+
+addOfNull
+SELECT CAST(NULL AS INT) + CAST(NULL AS FLOAT) AS n;
+
+n:d
+null
+;
+
+
+divOfNull
+SELECT 5 / CAST(NULL AS FLOAT) + 10 AS n;
```

Comment:
Why not `SELECT 5 / NULL AS n`, without casting?

Reply:
Because I want to check that the a cast preserve the null value and its associated type. I've renamed the test and added a basic one without cast.